fixtures/module/module.tf

```diff
@@ -6,7 +6,15 @@ module "test_module" {
   region = "${var.region}"
   env_name = "${var.env_name}"
 
+  env_name = "${var.env_name}"
```

Comment:
Do we need these variables/outputs in the module fixture? If not let's remove them.